### PR TITLE
updated azure instructions

### PR DESCRIPTION
**Pull request by DanielLanger**

### CODE REVIEW

File: content/integrations/azure.md [hunk 1]
```diff
@@ -18,25 +18,73 @@ sidebar:
...
-To setup Azure monitoring, go to [Azure integrations][1] and follow instructions on the page.
 
 ![Azure default dashboard](/static/images/azure.png)
 
 Learn more about how to monitor Azure performance metrics thanks to [our series of posts](https://www.datadoghq.com/blog/how-to-monitor-microsoft-azure-vms/). We detail the key performance metrics, how to collect them, and how to use Datadog to monitor Azure VMs.
 
 #### Enable diagnostics
-To enable agent-less monitoring, you must enable diagnostics. Right now this is only support by Windows based machines. To do this, first go to [Azure preview portal][2] then follow the instructions below.
+To advanced metric collection for virtual machines, you must enable diagnostics. Right now this is only support by Windows based machines. To do this, first go to [Azure preview portal][2] then follow the instructions below.
 ![](/static/images/azure_diag_manual.png)
```
Comment (irabinovitch) [inline content/integrations/azure.md]: Its no longer called he preview portal.  I believe its just "Azure Portal"

**Comment by irabinovitch** [comment]
Lets do a PR into staging first so folks can review. 

* do we have higher res images we can use some of them seem fuzzy?

File: content/integrations/azure.md (hunk 1 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: Yes correct. Is there a word missing in "To advanced metric collection for virtual machines"? @irabinovitch

**Comment by DanielLanger** [comment]
I don't know why they not high res, screen shots from chrome. Maybe a couple are blown up a little b/c the icon is small in the Azure portal(s)

File: content/integrations/azure.md (hunk 1 above)
Comment (irabinovitch) [inline content/integrations/azure.md]: "To advanced metric collection for virtual machines" looks to be missing a word.

File: content/integrations/azure.md [hunk 2]
```diff
@@ -18,25 +18,73 @@ sidebar:
 
 ### Overview
 
-Datadog's Azure integration currently monitors Virtual Machines (diagnostics must be enabled), and SQL Databases. For any other service, please reach out to [support@datadoghq.com][3].
+Datadog's Azure integration currently monitors Virtual Machines, SQL Databases, and App Services. For any other service, please reach out to [support@datadoghq.com][3].
 Datadog's agent can also be deployed on your Azure virtual machines.
```
Comment (irabinovitch) [inline content/integrations/azure.md]: We break each of these into their own docs page.

File: content/integrations/azure.md [hunk 3]
```diff
@@ -18,25 +18,73 @@ sidebar:
...
+#### Getting your tenant ID
+1. Go to [manage.windowsazure.com][5]
+2. Once the URL redirects, copy the tenant name from the URL. It is the text in between the @ and # symbol <img src="/static/images/azure/azure_tenant_url.png" style="width: 75%;display:block;"/>
+3. Paste the Tenant Name in the form on the Datadog setup page <img src="/static/images/azure/tenant_name_form.png" style="width: 50%;display:block;display:block;"/>
+
+#### Creating the Web Application
+
+1. Navigate to the “Active Directory” tab on the left-hand menu <img src="/static/images/azure/active_directory_icon.png" style="width: 25%;display:block;"/>
+2. Click on the Active Directory you want to monitor <img src="/static/images/azure/active_directory_name_click.png" style="width: 50%;display:block;"/>
+3. Click on "Applications" <img src="/static/images/azure/application_click.png" style="width: 50%;display:block;"/>
```
Comment (irabinovitch) [inline content/integrations/azure.md]: @technovangelist @jyee Do we prefer [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images) for including images over html?

File: content/integrations/azure.md [hunk 4]
```diff
@@ -18,25 +18,73 @@ sidebar:
...
+10. Repeat this process for any other subscriptions you would like to monitor
+
+**Navigate back to the Datadog setup page, and click "Update Configuration" to complete the installation process**
+
+**In a few minutes, metrics from applications under that subscription will begin to appear!**
+
 
-To setup Azure monitoring, go to [Azure integrations][1] and follow instructions on the page.
 
 ![Azure default dashboard](/static/images/azure.png)
```
Comment (irabinovitch) [inline content/integrations/azure.md]: This should read  call out "Azure VM Default Dashboard"  or move into a dedicated VM doc

File: content/integrations/azure.md [hunk 5]
```diff
@@ -18,25 +18,73 @@ sidebar:
...
 
 Learn more about how to monitor Azure performance metrics thanks to [our series of posts](https://www.datadoghq.com/blog/how-to-monitor-microsoft-azure-vms/). We detail the key performance metrics, how to collect them, and how to use Datadog to monitor Azure VMs.
```
Comment (irabinovitch) [inline content/integrations/azure.md]: "Learn more about how to monitor Azure VM performance metrics"

File: content/integrations/azure.md [hunk 6]
```diff
@@ -18,25 +18,73 @@ sidebar:
...
 #### Enable diagnostics
-To enable agent-less monitoring, you must enable diagnostics. Right now this is only support by Windows based machines. To do this, first go to [Azure preview portal][2] then follow the instructions below.
+To collect advanced metrics for virtual machines, you must enable diagnostics. Right now this is only support by Windows based machines. To do this, first go to [Azure Portal][2] then follow the instructions below.
 ![](/static/images/azure_diag_manual.png)
 
 After locating your VM:
+
 1. Click on the CPU percentage today panel to show metrics panel
 2. Click on Diagnostics
 3. Shift the switch to open
```
Comment (irabinovitch) [inline content/integrations/azure.md]: Can we provide links to information on automated steps for enabling this during provisioning? or via the api? Doing this one by one on hundreds or thousands of machines doesn't seem scale-able.

File: content/integrations/azure.md [hunk 7]
```diff
@@ -73,3 +121,4 @@ Please make sure you have enabled diagnostics on your VMs. Diagnostics allows VM
    [2]: https://portal.azure.com
    [3]: mailto:support@datadoghq.com
    [4]: https://management.azure.com
+   [5]: https://manage.windowsazure.com
```
Comment (irabinovitch) [inline content/integrations/azure.md]: No metrics csv data here?

File: content/integrations/azure.md (hunk 3 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: Had sizing/display issues with markdown language for images FYI

File: content/integrations/azure.md (hunk 4 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: I will replace with the new Azure Overview Dashboard

File: content/integrations/azure.md (hunk 6 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: @irabinovitch @tmichelet Do we have links to these instructions?

File: content/integrations/azure.md (hunk 3 above)
Comment (technovangelist) [inline content/integrations/azure.md]: yes, i would prefer using markdown for everything. the particular flavor of markdown used is kramdown

**Comment by technovangelist** [comment]
since you are making significant changes to the structure of the doc, can you update it to use the standard format?

File: content/integrations/azure.md (hunk 7 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: what do you mean?

Comment (technovangelist) [inline content/integrations/azure.md]: take a look at https://github.com/DataDog/documentation/blob/master/content/integrations/elasticsearch.md, scroll to the bottom. there is a block that says: <%= get_metrics_from_git()%>. This uses whatever git_integration_title is set to in the frontmatter at the top to grab the metrics csv from dogweb

File: content/integrations/azure.md (hunk 3 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: Is there a way to add inline style using kramdown? I used html to easily size and position the images @technovangelist

File: content/integrations/azure.md (hunk 7 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: gotcha. tried a few names, but all came back nil. @tmichelet, any idea what the git_integration_title is for azure?

Comment (irabinovitch) [inline content/integrations/azure.md]: @DanielLanger let us know if you need help with this one, but it should just be adding that macro matt provided at the bottom.

File: content/integrations/azure.md (hunk 2 above)
Comment (irabinovitch) [inline content/integrations/azure.md]: @DanielLanger thoughts?

Comment (DanielLanger) [inline content/integrations/azure.md]: @irabinovitch Definitely at some point. With WPC in two days I won't have the time to create them before hard.

File: content/integrations/azure.md (hunk 7 above)
Comment (DanielLanger) [inline content/integrations/azure.md]: @irabinovitch talked to @tmichelet about it. We moved the azure metadata to each sub integration, so the overall azure integration isn't associated w/ a specific CSV file

File: content/integrations/azure.md (hunk 3 above)
Comment (irabinovitch) [inline content/integrations/azure.md]: Doesn't look like this got addressed before merging.  Please work with @technovangelist or @jyee  in #documentation to get this addressed.

File: content/integrations/azure.md (hunk 7 above)
Comment (irabinovitch) [inline content/integrations/azure.md]: @DanielLanger Doesn't look this like was addressed. All docs must have metadata in them.

**Comment by irabinovitch** [comment]
@DanielLanger in the future, best to squash before merging.

**Pull request merged**